terraform/20-app/cloud-front.front-end.tf

```diff
@@ -1,5 +1,6 @@
 locals {
-  fifteen_minutes_in_seconds = 900
+  fifteen_minutes_in_seconds         = 900
+  managed_caching_disabled_policy_id = "4135ea2d-6df8-44a3-9df3-4b5a84be39ad"
```

Comment:
This is the managed caching disabled policy from AWS

- https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-cache-policies.html#managed-cache-policy-caching-disabled